tmt/steps/provision/mrack.py

```diff
@@ -197,6 +206,34 @@ class MrackHWNotGroup(MrackHWGroup):
     name: str = 'not'
 
 
+def _get_custom_config() -> dict[str, Any]:
+    if TMT_MRACK_HW_FILTERS_FILE.is_absolute():
+        config_content = TMT_MRACK_HW_FILTERS_FILE.read_text()
+    else:
+        config_content = (tmt.utils.Config().path / TMT_MRACK_HW_FILTERS_FILE).read_text()
+    return tmt.utils.yaml_to_dict(config_content)
```

Comment:
Right, I would not let user override just this particular file when there's a nice envvar for defining where tmt configuration lives. So this should always use `Config.path`, plus the file name mentioned above.

Reply:
Got it, will update accordingly:)